deny.toml

```diff
@@ -0,0 +1,15 @@
+[advisories]
+unsound = "deny"
+yanked = "deny"
+unmaintained = "deny"
+
+[licenses]
+allow-osi-fsf-free = "either"
+copyleft = "warn"
+allow = ["MPL-2.0"]
```

Comment:
`MPL-2.0` is considered `copyleft` but as far as I can tell it avoids the issues associated with viral licenses like the GPL. @complexspaces do you have some kind of policy on what licenses are allowed in 1P dependencies?

Reply:
We do, but we made the same observation about the MPL, so it is on our internal allow-list. I don't see any issues with allowing it here either 👍.